src/screens/sync/Devices.tsx

```diff
@@ -207,17 +227,19 @@ export const Devices = ({ mode }: { mode: ViewMode }) => {
   );
 
   return (
-    <View style={{ paddingVertical: spacing.large }}>
-      {status === "loading" ? (
-        <View>
-          <Text size="large" bold textAlign="center">
-            {t(m.searching)}
-            <AnimatedEllipsis />
-          </Text>
-        </View>
-      ) : mode === "list" ? (
-        <DeviceList />
-      ) : null}
-    </View>
+    <React.Fragment>
+      <View style={{ paddingVertical: spacing.large }}>
+        {status === "loading" ? (
+          <View>
+            <Text size="large" bold textAlign="center">
+              {t(m.searching)}
+              <AnimatedEllipsis />
+            </Text>
+          </View>
+        ) : (
+          children
+        )}
+      </View>
+    </React.Fragment>
```

Comment:
nit: no need for fragment here

Reply:
https://github.com/digidem/mapeo-mobile-shell/pull/28/commits/f63808067cd2927413d7e868415aa0a2fcafec8c

I originally added the modal here, and forgot to take out the Fragment when I moved it out.